testing/percona-xtrabackup/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtrabackup
+pkgver=2.4.6
+pkgrel=0
+pkgdesc='Percona XtraBackup is a MySQL non-blocking backup solution for InnoDB and XtraDB data'
+arch=all
+url='http://www.percona.com/software/percona-xtrabackup/'
+license='GPL'
+makedepends='bash bison cmake ncurses-dev libaio-dev libgcrypt-dev zlib-dev curl-dev libev-dev vim'
+source="https://github.com/percona/percona-xtrabackup/archive/percona-xtrabackup-$pkgver.tar.gz
+        fix-posix_timers.patch
+        "
+
+subpackages="$pkgname-doc $pkgname-test:mytest
+        "
+
+_builddir="$srcdir/percona-xtrabackup-percona-xtrabackup-$pkgver"
```

Comment:
please use `builddir` instead of `_builddir` and remove the `prepare()` function. If builddir is set then will abuild apply the patches itself.